ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigCapella.java

```diff
@@ -35,4 +35,8 @@ static SpecConfigCapella required(SpecConfig specConfig) {
 
   @Override
   Optional<SpecConfigCapella> toVersionCapella();
+
+  UInt64 getMaxWithdrawalsPerPayload();
+
+  UInt64 getMaxBlsToExecutionChanges();
```

Comment:
I suspect both of these are better off just being `int` given they're known small numbers. Will save a bunch of garbage and extracting ints from UInt64.